understanding/understanding-techniques.html

```diff
@@ -40,14 +40,14 @@ <h1>Understanding Techniques for WCAG Success Criteria</h1>
 		<p><a href="conformance">Understanding Conformance</a> provides related information, including on <a href="conformance#accessibility-support">understanding accessibility support</a>.</p>
 		<section>
 			<h2>Techniques are Informative</h2>
-			<p><em>Techniques are informative—that means they are not required. The basis for determining conformance to WCAG 2.1 is the success criteria from the WCAG 2.1 standard—not the techniques.</em></p>
-			<p class="prefix"><em>Note 1: </em>W3C cautions against requiring W3C's sufficient techniques. The only thing that should be required is meeting the WCAG 2.1 success criteria. To learn more, see:</p>
+			<p><em>Techniques are informative—that means they are not required. The basis for determining conformance to WCAG 2 are the success criteria from the WCAG 2 standard—not the techniques.</em></p>
```

Comment:
```suggestion
			<p><em>Techniques are informative — that means they are not required. The basis for determining conformance to WCAG 2 are the success criteria from the WCAG 2 standard — not the techniques.</em></p>
```

Matching the styling for em-dash interruptions that I noted in the `understanding-act-rules.html` page text. Having space between the words and em-dashes is also the formatting used on the `conformance.html` page.

Not a big deal either way, though, this is mainly a consistency thing.